### PR TITLE
[jest-config-terra] Add png to the proxy in moduleNameMapper

### PR DESCRIPTION
**Pull request by ryanthemanuel**
### Summary

Add png to the proxy in moduleNameMapper

### CODE REVIEW

**Review by mjhenkes — APPROVED** [review]

**Review by benbcai — APPROVED** [review]
With updated changelog

**Review by mjhenkes — CHANGES_REQUESTED** [review]
you'll have to release terra-functional testing now too

**Comment by benbcai** [comment]
There is a change in terra-open-source-script that needs to be released. 

This entry in the changelog can be removed: https://github.com/cerner/terra-toolkit/pull/650/files#diff-7e77c8894b82f9d3dca674c028176de28d045adf5c6c187addd64da4c5147144R8

**Review by mjhenkes — APPROVED** [review]

File: packages/terra-toolkit-docs/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 1.12.0 - (May 7, 2021)
+
+* Changed
+  * Minor dependency version bump
+
```
Comment (ryanthemanuel) [inline packages/terra-toolkit-docs/CHANGELOG.md]:
```suggestion
```

**Review by benbcai — APPROVED** [review]

**Pull request merged**